src/main/java/frc/robot/autoCommands/autoSetFlywheel.java

```diff
@@ -0,0 +1,68 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.autoCommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.subsystems.interfaces.ShooterInterface;
+
+public class autoSetFlywheel extends CommandBase {
+
+  ShooterInterface shooter;
+  private double velocity;
+  private double maxVelocity;
+
+  /**
+   * Creates a new autoSetFlywheel.
+   */
+  public autoSetFlywheel(ShooterInterface shooter, double velocity, double maxVelocity) {
+    this.shooter = shooter;
+    this.velocity = velocity;
+    this.maxVelocity = maxVelocity;
+    addRequirements((SubsystemBase)shooter);
+  }
+
+  public autoSetFlywheel(ShooterInterface shooter, double velocity) {
+    this(shooter, velocity, shooter.getMaximumFlywheelSpeed());
+  }
+
+  public static autoSetFlywheel autoFlywheelMax(ShooterInterface shooter) {
+    return new autoSetFlywheel(shooter, shooter.getMaximumFlywheelSpeed(), shooter.getMaximumFlywheelSpeed());
+  }
+
+  public static autoSetFlywheel autoFlywheelOff(ShooterInterface shooter) {
+    return new autoSetFlywheel(shooter, 0.0, shooter.getMaximumFlywheelSpeed());
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (velocity <= maxVelocity){
+      shooter.setFlywheelSpeed(velocity);
```

Comment:
could replace this using Java.lang.math.min()
shooter.setFlywheelSpeed(min(velocity,maxVelocity))